src/main/java/frc/team2412/robot/RobotMap.java

```diff
@@ -1,12 +1,45 @@
 package frc.team2412.robot;
 
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+
 //This is the class in charge of all the motors, motor ids, and any other sensors the robot uses. 
 //remember to declare robot container at the bottom of this class 
 public class RobotMap {
 
 	// IDs
 	public static int exampleID = 1;
-
+	public static int indexBackMotorID = 1;
+	public static int indexFrontMotorID = 2;
+	public static int indexMidMotorID = 3;
+	public static int BBSensorID = 1;//Back Back
+	public static int BTSensorID = 2;//Back Tower
+	public static int TSensorID = 3; //Middle Position / Tower
+	public static int FTSensorID = 4;//Front Tower
+	public static int FFSensorID = 5;//Front Front
```

Comment:
The IDs should be `private final static int` as only this class ever uses them & they are immutable.